### PR TITLE
Cloudwatch: Update AWS DynamoDB Metrics

### PR DESCRIPTION
**Pull request by LeonardoBoleli**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: pkg/tsdb/cloudwatch/constants/metrics.go [hunk 1]
```diff
@@ -395,7 +395,7 @@ var NamespaceMetricsMap = map[string][]string{
...
-	"AWS/Events":                         {"DeadLetterInvocations", "Events", "FailedInvocations", "IngestionToInvocationStartLatency", "Invocations", "InvocationsFailedToBeSentToDlq", "InvocationsSentToDlq", "MatchedEvents", "ThrottledRules", "TriggeredRules"},
+	"AWS/Events":                         {"DeadLetterInvocations", "Events", "FailedInvocations", "Invocations", "InvocationAttempts", "InvocationsCreated", "InvocationsFailedToBeSentToDlq", "IngestiontoInvocationCompleteLatency", "IngestiontoInvocationStartLatency", "InvocationsSentToDlq", "MatchedEvents", "RetryInvocationAttempts", "SuccessfulInvocationAttempts", "ThrottledRules", "TriggeredRules", "PutEventsApproximateCallCount", "PutEventsApproximateFailedCount", "PutEventsApproximateSuccessCount", "PutEventsApproximateThrottledCount", "PutEventsEntriesCount", "PutEventsFailedEntriesCount", "PutEventsLatency", "PutEventsRequestSize", "PutPartnerEventsApproximateCallCount", "PutPartnerEventsApproximateFailedCount", "PutPartnerEventsApproximateThrottledCount", "PutPartnerEventsApproximateSuccessCount", "PutPartnerEventsEntriesCount", "PutPartnerEventsFailedEntriesCount", "PutPartnerEventsLatency"},
```
Comment (njvrzm) [inline pkg/tsdb/cloudwatch/constants/metrics.go:398]: Please sort all the metric names here for consistency. (I know not all the entries are sorted, but they're meant to be 😄 )

**Review by njvrzm — CHANGES_REQUESTED** [review]
One small change request.

**Comment by njvrzm** [comment]
Hi @LeonardoBoleli, and thanks for your contribution! I've requested one minor change. Also if you could add a little bit of information to the pull request for the record that would be great.

File: pkg/tsdb/cloudwatch/constants/metrics.go (hunk 1 above)
Comment (LeonardoBoleli) [reply pkg/tsdb/cloudwatch/constants/metrics.go]: Done!

**Review by iwysiu — APPROVED** [review]
Thanks for pr!

link to aws docs:
- [for the events metrics](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-monitoring.html)
- [for dynamodb metrics](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/metrics-dimensions.html)

**Review by njvrzm — APPROVED** [review]
👍🏻

**Pull request merged**